package.json

```diff
@@ -31,7 +31,7 @@
   },
   "scripts": {
     "compile": "tsc",
-    "lint": "tslint -c tslint.json src/**.ts tests/**.ts",
+    "lint": "tslint -c tslint.json -p .",
```

Comment:
What does the flag `-p` stand for?

Reply:
It tells tslint to start in the current directory

```
-p, --project:
    The path or directory containing a tsconfig.json file that will be
    used to determine which files will be linted. This flag also enables
    rules that require the type checker.
```
https://palantir.github.io/tslint/usage/cli/